src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -8896,19 +8873,16 @@ void VRFCondenserEquipment::SizeVRFCondenser(EnergyPlusData &state)
 
     static constexpr std::string_view RoutineName("SizeVRFCondenser");
 
-    int PltSizCondNum;         // Plant Sizing index for condenser loop
     Real64 rho;                // local fluid density [kg/m3]
     Real64 Cp;                 // local fluid specific heat [J/kg-k]
     Real64 tmpCondVolFlowRate; // local condenser design volume flow rate [m3/s]
-    bool ErrorsFound;          // indicates problem with sizing
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:8899]:(style),[variableScope],The scope of the variable 'PltSizCondNum' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:8903]:(style),[variableScope],The scope of the variable 'ErrorsFound' can be reduced.